integration-tests/ahm/src/mock.rs

```diff
@@ -171,9 +172,10 @@ pub fn enqueue_dmp(msgs: Vec<InboundDownwardMessage>) {
 	}
 }
 
-/// Enqueue DMP messages on the parachain side.
-pub fn enqueue_ump(msgs: Vec<UpwardMessage>) {
-	for msg in msgs {
+/// Enqueue UMP messages on the Relay Chain side.
```

Comment:
What does the block number mean?

Reply:
I added a doc there - The block number parameter indicates the block at which these messages were sent. It may be set to zero when the block number information is not important for a test.